pkgs/games/0ad/game.nix

```diff
@@ -43,6 +43,10 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+     sed -e 's|ENSURE|//ENSURE|g' -i source/lib/sysdep/arch/x86_x64/cache.cpp
```

Comment:
I will replace this with a rebased upstream patch in a bit.

Reply:
Meh, doesn't rebase easily. Let's stick to that hack until 0.24, which is expected around the end of january.